### DISCUSSION TITLE
configurable pre defined windows setup

### DISCUSSION THREAD

**Discussion by ilan-pinto** [rxhanson/Rectangle#795] [Ideas]
usually, I have a regular setup of windows e.g: 

- 1/2 left side with chrome 
- 1/4 top right  - zoom 
- 1/4 bottom right - WhatsApp desktop  
I would like to have a shortcut that resizes the windows bringing the above setup.

ultimately being able to predefine 5 setup could be good

**Comment by rxhanson** [comment]
Thanks for the feedback. This functionality is built into [Rectangle Pro](https://rectangleapp.com/pro), and I don't have plans to roll it into Rectangle. The reasoning is primarily to keep Rectangle simple. 

In Rectangle Pro, the easiest way to do this is to organize your apps the way you want them, then go to the `Applications` tab and add a group. You'll be able to select the current layout as a group, and add a keyboard shortcut for it.
